### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by OR13**
Add instructions for downloading report json

### CODE REVIEW

**Review by mprorock — APPROVED** [review]
This is a big need as then we can setup automated visualization around the test results in a meaningful way

File: README.md [hunk 1]
```diff
@@ -7,11 +7,31 @@ The VC HTTP API repository contains a standard API specification for constructin
...
+  - Architecture for the issuer model is discussed in the [architecture document](./docs/architecture.md)
+  - Within docs, the versions of the api are stored in the appropriate folder under [versions](./docs/versions)
+  - The core API itself is detailed in the [spec file](./docs/vc-http-api.yml) and this spec should be used as the master reference for the API
 - [packages](./packages/) which contains test suites and related utilities for integration testing, test suite results, and related assets.
 
+### Data Visualization
+
+You may wish to programatically download the latest test results as JSON.
+
+#### All Vendors Test Results
```
Comment (TallTed) [inline README.md:19]:
```suggestion
#### Test Results for All Implementations 
```

File: README.md [hunk 2]
```diff
@@ -7,11 +7,31 @@ The VC HTTP API repository contains a standard API specification for constructin
...
+
+- HTML Report: TODO
+
+```
+curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/testResults.json | jq
+```
+
+#### A specific vendor's Test Results
```
Comment (TallTed) [inline README.md:27]:
```suggestion
#### Test Results for a Specific Implementation
```

File: README.md [hunk 3]
```diff
@@ -7,11 +7,31 @@ The VC HTTP API repository contains a standard API specification for constructin
...
+
+- HTML Report: https://w3c-ccg.github.io/vc-http-api/test-suite/VENDOR_NAME
+
+```
+curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/VENDOR_NAME/testResults.json | jq
```
Comment (TallTed) [inline README.md:32]:
```suggestion
- HTML Report: https://w3c-ccg.github.io/vc-http-api/test-suite/IMPLEMENTATION_NAME

```
curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/IMPLEMENTATION_NAME/testResults.json | jq
```

**Comment by TallTed** [comment]
In all seriousness -- consider a LARGE vendor with a LARGE product portfolio with smaller product implementation teams, which do not cross-develop, because (for instance) the products are written in different languages for different platforms, etc.  It's entirely possible that vendor may have SEVERAL DID implementations across that product portfolio, and wish to record results for each implementation.

Test results should NOT focus on the VENDOR.

File: README.md [hunk 4]
```diff
@@ -7,11 +7,31 @@ The VC HTTP API repository contains a standard API specification for constructin
 The vc-http-api repository is structured with two main sections:
 
 - [docs](./docs/) which contains the VC HTTP API spec itself in [OpenAPI](https://swagger.io/specification/) (formerly known as Swagger) form as well as related documentation to assist in building and testing a compliant implementation of the API spec
-  * Architecture for the issuer model is discussed in the [architecture document](./docs/architecture.md)
-  * Within docs, the versions of the api are stored in the appropriate folder under [versions](./docs/versions)
-  * The core API itself is detailed in the [spec file](./docs/vc-http-api.yml) and this spec should be used as the master reference for the API 
+  - Architecture for the issuer model is discussed in the [architecture document](./docs/architecture.md)
+  - Within docs, the versions of the api are stored in the appropriate folder under [versions](./docs/versions)
```
Comment (TallTed) [inline README.md:11]:
```suggestion
  - Within docs, the versions of the API are stored in the appropriate folder under [versions](./docs/versions)
```

File: README.md [hunk 5]
```diff
@@ -7,11 +7,31 @@ The VC HTTP API repository contains a standard API specification for constructin
...
+
+You may wish to programatically download the latest test results as JSON.
+
+#### Test Results for All Implementations
+
+- HTML Report: TODO
+
+```
+curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/testResults.json | jq
+```
```
Comment (TallTed) [inline README.md:25]: This nests the codeblock within the bullet

```suggestion
  ```
  curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/testResults.json | jq
  ```
```

File: README.md [hunk 6]
```diff
@@ -7,11 +7,31 @@ The VC HTTP API repository contains a standard API specification for constructin
...
+
+#### Test Results for a Specific Implementation
+
+- HTML Report: https://w3c-ccg.github.io/vc-http-api/test-suite/IMPLEMENTATION_NAME
+
+```
+curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/IMPLEMENTATION_NAME/testResults.json | jq
+```
```
Comment (TallTed) [inline README.md:33]: This nests the codeblock within the bullet

```suggestion
  ```
  curl -s https://w3c-ccg.github.io/vc-http-api/test-suite/IMPLEMENTATION_NAME/testResults.json | jq
  ```
```

**Pull request merged**